.travis.yml

```
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "6.10"
+  - "12.18.4"
```

Please use the following:

```
- "10"
- "12"
- "latest"
```

---

Okay, I did! ("latest" is not good, it causes failed suddenly in the future.)

---

huh...? I don't think this is what @DanielRuf mean... He mean to use three of them at once on travis.yml? 😅 

IMO it's okay if `latest` is failing, with that we will immediately know that the latest version of Node.js have problem when building materialize.

---

"fail early" is what we want, so `latest` should be added.

And `10` too.

---

> He mean to use three of them at once on travis.yml?

Correct, all three at once (a build matrix).

> IMO it's okay if latest is failing, with that we will immediately know that the latest version of Node.js have problem when building materialize.

Exactly.

---

@Smankusors 
I understand now, thanks for your comment! ❤️  Fixed!

---

It seems nvm doesn't have this tag anymore. Can you try `node` and `*` instead of `latest`?

---

@DanielRuf 
Thank you! Yes, I just did it and succeeded!